crates/repository/src/lib.rs

```diff
@@ -170,9 +164,32 @@ impl Repository {
 
         println!("Running: {cargo_command}");
 
+        let shell_command = if target == "nao" {
+            if OS_IS_NOT_LINUX {
```

Comment:
move the `use_docker` variable to its parent scope and use it here. This branch should depend on whether we want to use docker, and not whether the OS is not linux. I imagine actively selecting for the docker build in some future feature

Reply:
Thanks, done.